packages/mask/dashboard/utils/regexp.ts

```diff
@@ -1,5 +1,5 @@
 export const passwordRegexp = /^(?=.{8,20}$)(?=.*[A-Z])(?=.*[a-z])(?=.*\d)(?=.*[^\dA-Za-z]).*/
 
-export const emailRegexp = /^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/
+export const emailRegexp = /^([\w!#$%&*+./=?^`{|}~’-]{1,64}@([\dA-Za-z-]{1,255}.[\dA-Za-z-]{2,}))$/
```

Comment:
https://beesbuzz.biz/code/439-Falsehoods-programmers-believe-about-email
@guanbinrui ^_^

Reply:
Actually, I would suggest using browser's input type=email to validate it, even if your RegEx looks right even for some extreme cases

Reply:
We need to validate the email to display the helper text.